Navbar test

mfear-0 opened the pull request.

AricaLConrad commented:
I'm not entirely sure of the impact of your changes, as I can't quite picture what the navigation menu will look like until it's on my end. I did look through all of your changes, though. You can probably merge back to main when you are ready.

It seems like you commented out the placeholder titles I had made (which is absolutely fine). Do you want me to remove those placeholder titles completely now? 

Also, I noticed you deleted some of my comments. Out of curiosity, were there any problems with my comments?

mfear-0 commented:
> Also, I noticed you deleted some of my comments. Out of curiosity, were there any problems with my comments?

No, sorry, I ended up deleting and moving a bunch of stuff on accident. I tried to only remove ones that said like "this is just a placeholder" since i replaced the placeholder code, but if i replaced other comments i didn't mean to.

AricaLConrad commented:
Alright, that makes sense. Thank you for the reply.
